package.json

```
@@ -3,7 +3,7 @@
   "version": "3.1.0",
   "private": true,
   "scripts": {
-    "postinstall": "npx patch-package && npx jetify"
+    "postinstall": "npx --no-install patch-package && npx --no-install jetify"
```

Shouldn't this suffice?

```suggestion
    "postinstall": "patch-package && jetify"
```

---

Last I remember, there were issues doing this on windows. cc @jobvs

---

Exactly why we are using npx aside of that being part of the official documentation of the tools

---

Ok, seems a bit strange. As far as I know you never need to use npx in npm scripts when the dependency is in package.json. Patch-package also doesn't use npx in its docs.

It seems to work fine here on Windows with latest node when I remove npx from the script.

---

As far as I know npx is executing executable scripts from ./bin folder. It's an unlucky side effect that it installs dependencies in a temp folder. E.g. you can't just run jetify in the cmd it does nothing; npx will. 

But indeed the scripts work fine if called directly in the post install script.